synapse/lib/aha.py

```diff
@@ -555,6 +644,128 @@ async def addAhaSvc(self, name, info, network=None):
         # mostly for testing...
         await self.fire('aha:svcadd', svcinfo=svcinfo)
 
+    def _getAhaName(self, name):
+        # the modern version of names is absolute or ...
+        if name.endswith('...'):
+            netw = self.conf.get('aha:network')
+            name = name[:-2] + netw
+        return name
+
+    async def getAhaPool(self, name):
+        return self._getAhaPool(name)
+
+    def _getAhaPool(self, name):
```

Comment:
Bit of a nitpick but `_getAhaPool` doesn't seem necessary, `getAhaPool` is the only place that needs it and can just do a `_getAhaName` before calling `_loadPoolInfo`

Reply:
Did that to allow an uninterruptible version via no `async` but seems unnecessary now. will remove :+1: